mm/2s2h/Rando/CheckTracker/CheckTracker.cpp

```diff
@@ -36,10 +46,27 @@ static s32 sScrollToTargetScene = -1;
 static s32 sScrollToTargetEntrance = -1;
 static ImGuiTextFilter sCheckTrackerFilter;
 
+static ImVec2 imageSize;
+static ImVec4 colorFilter;
+
 std::map<SceneId, std::vector<RandoCheckId>> sceneChecks;
 std::vector<SceneId> sortedSceneIds;
 std::unordered_map<RandoCheckId, std::string> readableCheckNames;
 
+std::vector<std::pair<const char*, ImVec4>> checkTypeIconMap = {
+    /*RCTYPE_UNKNOWN*/ { gItemIconBombersNotebookTex, { 1, 1, 1, 1 } },
+    /*RCTYPE_CHEST*/ { gItemIconBombersNotebookTex, { 1, 1, 1, 1 } },
+    /*RCTYPE_COW*/ { gItemIconRomaniMaskTex, { 1, 1, 1, 1 } },
+    /*RCTYPE_FREESTANDING*/ { gItemIconBombersNotebookTex, { 1, 1, 1, 1 } },
+    /*RCTYPE_NPC*/ { gItemIconBombersNotebookTex, { 1, 1, 1, 1 } },
+    /*RCTYPE_OWL*/ { gWorldMapOwlFaceTex, { 1, 1, 1, 1 } },
+    /*RCTYPE_POT*/ { gItemIconBombersNotebookTex, { 1, 1, 1, 1 } },
+    /*RCTYPE_SHOP*/ { gRupeeCounterIconTex, { 0.78f, 1, 0.39f, 1 } },
```

Comment:
will we ever have another case we don't want full white? may be better to just handle this in the code than store it in the map